src/ElmSharp/ElmSharp/EcoreMainloop.cs

```diff
@@ -32,7 +32,9 @@ public static class EcoreMainloop
 
         static Interop.Ecore.EcoreTaskCallback _nativeHandler;
 
-        static EcoreMainloop()
+        
+
```

Comment:
Initialize all static fields in 'EcoreMainloop' when those fields are declared and remove the explicit static constructor